multiqc/modules/sortmerna/sortmerna.py

```diff
@@ -33,14 +34,14 @@ def __init__(self):
             log.debug("Could not find any SortMeRNA data in {}".format(config.analysis_dir))
             raise UserWarning
 
-        log.info("Found {} logs".format(len(self.sortmerna)))
```

Comment:
Did you mean to drop these two lines? I was only referring the to the `log.debug` statement that looked like it was accidentally left over from testing..